build.sbt

```diff
@@ -2,28 +2,26 @@
  * Copyright © 2016-2017 Lightbend, Inc. <http://www.lightbend.com>
  */
 
-lazy val `sbt-javaagent` = project in file(".")
-
-sbtPlugin := true
-
-name := "sbt-javaagent"
-organization := "com.github.sbt"
-
 // sbt cross build
-crossSbtVersions := Seq("0.13.18", "1.2.8")
+crossSbtVersions := Seq("1.2.8")
```

Comment:
**suggestion:** bump this and you probably won't have to set `build.properties` in all the tests.

Reply:
good catch!

Reply:
Likely on old sbt versions to make sure it's compatible since that version, and doesn't depend on anything from newer sbt versions. But should be no problem to just bump to latest now.